lib/tasks/migrations/migrations.rake

```diff
@@ -255,8 +259,6 @@ namespace :migrations do
       "invite"
     when "2"
       "approval"
-    else
-      "open"
```

Comment:
Why are we dropping this?

Reply:
Good catch, this is a copy/paste mistake.